field/FieldCreate.F90

```diff
@@ -89,7 +89,7 @@ subroutine field_empty_complete( field, &
 
       bounds = make_bounds(num_levels=num_levels, ungridded_dims=ungridded_dims)
       call ESMF_FieldEmptyComplete(field, typekind=typekind, &
-           gridToFieldMap=gridToFieldMap, &
+!#           gridToFieldMap=gridToFieldMap, &
```

Comment:
Delete if not needed.

Reply:
Yeah - not needed now.  Probably will be needed soon.

Reply:
Please do these with the +/- suggestion. Makes it much easier to address.